### PR TITLE
Updating readmes

### PR DESCRIPTION
**Pull request by briancaine**
https://github.com/skycoin/exchange-api/issues/11
https://github.com/skycoin/exchange-api/issues/12

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,7 +17,11 @@
   - Getting balance
 
  Node using jsonrpc 2.0 protocol for rpc calls:
- ## RPC methods:
+
+## RPC methods:
+
+More details on the RPC interface are available in [the RPC README file](https://github.com/skycoin/exchange-api/tree/master/rpc#readme).
```
Comment (gz-c) [inline README.md]: the link can be `(rpc/README.md)`

File: README.md [hunk 2]
```diff
@@ -33,6 +38,9 @@
   rpc endpoints for each exchange are different -  
   http://rpcaddr:rpcport/c2cx and http://rpcaddr:rpcport/cryptopia, respectively 
 # Cli
+
+More details on the CLI are available in [the CLI README file](https://github.com/skycoin/exchange-api/tree/master/cli#readme).
```
Comment (gz-c) [inline README.md]: the link can be `(cli/README.md)`

File: README.md [hunk 3]
```diff
@@ -33,6 +38,9 @@
...
+
  Cli command structure:
```
Comment (gz-c) [inline README.md]: If you're going to keep cli/README.md then why is there more documentation about the CLI in the main readme?

**Review by gz-c — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (briancaine) [reply README.md]: Fixed.

File: README.md (hunk 3 above)
Comment (briancaine) [reply README.md]: Yeah, I just noticed the `cli/readme.md` file was redundant. Removed.

**Pull request merged**